src/pages/Auth/SignupPage/Steps/StudentDetailStep/index.tsx

```diff
@@ -49,7 +51,8 @@ function StudentDetail({ onNext, onBack }: VerificationProps) {
   const studentNumber = (useWatch({ control, name: 'student_number' }) ?? '') as string;
 
   const [isCorrectId, setIsCorrectId, setInCorrectId] = useBooleanState(false);
-  const [isCorrectNickname, setIsCorrectNickname, setIsInCorrectNickname] = useBooleanState(false);
+  const [isCorrectNickname, setIsCorrectNickname, setInCorrectNickname] = useBooleanState(false);
+  const [, setIsCorrectEmail, setInCorrectEmail] = useBooleanState(false);
```

Comment:
```suggestion
  const [isCorrectNickname, setNicknameTrue, setNicknameFalse] = useBooleanState(false);
  const [, setEmailTrue, setEmailFalse] = useBooleanState(false);
```
이렇게 사용하는 것처럼(더 좋은 변수명이 떠오르지 않네요..) 변수명을 직관적으로 지으면 좋을 거 같아요.
현재 `setIsCorrectNickname` 자체는 `isCorrectNickname`을 true로 변경한다는게 바로 보이지 않는 것 같아요.

Reply:
저도 지금 당장 좋은 변수명이 떠오르지 않아, 네이밍 컨벤션 수정하면서 해당 변수명도 함께 수정하도록 하겠습니다 !